sagemaker-pipeline-multi-model/sagemaker-pipeline/pipelines/restate/pipeline.py

```diff
@@ -104,7 +104,7 @@ def get_pipeline(
     role=None,
     default_bucket=None,
     model_package_group_name="restatePackageGroup",  # Choose any name
-    pipeline_name="restate-p-roj2jxb1j0eu",  # You can find your pipeline name in the Studio UI (project -> Pipelines -> name)
+    pipeline_name="restate-p-XXXXXXXXX",  # You can find your pipeline name in the Studio UI (project -> Pipelines -> name)
```

Comment:
Is it required for the customer to fill in this pipeline name? If so, the preferred style is to use brackets, like `"<restate-p-XXXXXXXXX>"`, and clearly state this in the inline comment - suggested to revise from `You can find...` to `Find...` so it's clearer that this isn't optional.

Reply:
These are default arguments only as the actual values are passed from the Jupyter notebook. I have removed these references to avoid confusion and confirmed the pipeline still works even with the default arguments being empty. Change is in commit https://github.com/aws/amazon-sagemaker-examples/pull/3488/commits/a90d462d887dd1fe9bd50e908e687acbf42799dd. Thank you!